test_infra/src/lib.rs

```diff
@@ -1205,13 +1207,20 @@ impl<'a> GuestCommand<'a> {
         self
     }
 
+    pub fn set_print_cmd(&mut self, print_cmd: bool) -> &mut Self {
+        self.print_cmd = print_cmd;
+        self
+    }
+
     pub fn spawn(&mut self) -> io::Result<Child> {
-        println!(
-            "\n\n==== Start cloud-hypervisor command-line ====\n\n\
-                 {:?}\n\
-                 \n==== End cloud-hypervisor command-line ====\n\n",
-            self.command
-        );
+        if self.print_cmd {
```

Comment:
No problem with this. Just wondering why you don't want to output it for the performance tests?

Reply:
This is because we don't capture the output from the the performance tests while enabling this `print_cmd` would generate excessive log. We can't enable these output from the tests if we want to keep the output of `performance-metrics` readable.